io_mesh_w3d/common/utils/mesh_export.py

```diff
@@ -67,6 +67,10 @@ def retrieve_meshes(context, hierarchy, rig, container_name, force_vertex_materi
                         if pivot.name == mesh_object.vertex_groups[vertex.groups[1].group].name:
                             vert_inf.xtra_idx = index
                     vert_inf.xtra_inf = vertex.groups[1].weight
+
+                if vert_inf.bone_inf < 0.01 and vert_inf.xtra_inf < 0.01:
```

Comment:
Is this epsilon arbitarly? E.g. could be 0.00999 for 0.01 aswell

Reply:
yes

Reply:
this is just a hack for vanilla bfme files where both influcences are 0, also if the weight painting was done a bit unclean in blender such things can happen